tests/core/dom/range/getclientrects.js

```diff
@@ -0,0 +1,52 @@
+/* bender-tags: editor,dom,range */
+
+( function() {
+	'use strict';
+
+	var doc = CKEDITOR.document,
+		tests = {
+			setUp: function() {
+				this.playground = doc.getById( 'playground' );
+			},
+
+			'test simple single line selection': function() {
```

Comment:
Am I missing something or is there only one unit test? There are much more cases to be covered.